Xwt.WPF/Xwt.WPFBackend/FontBackendHandler.cs

```diff
@@ -76,20 +86,39 @@ public override IEnumerable<string> GetInstalledFonts ()
 
 		public override object Create (string fontName, double size, FontStyle style, FontWeight weight, FontStretch stretch)
 		{
+			FontFamily fontFamily;
+			if (!registeredFonts.TryGetValue (fontName, out fontFamily)) {
+				fontFamily = new FontFamily (fontName);
+			}
+
 			size = GetPointsFromDeviceUnits (size);
-			return new FontData (new FontFamily (fontName), size) {
+			return new FontData (fontFamily, size) {
 				Style = style.ToWpfFontStyle (),
 				Weight = weight.ToWpfFontWeight (),
 				Stretch = stretch.ToWpfFontStretch ()
 			};
 		}
 
-		[System.Runtime.InteropServices.DllImport ("gdi32.dll")]
-		static extern int AddFontResourceEx (string lpszFilename, uint fl, System.IntPtr pdv);
-
 		public override bool RegisterFontFromFile (string fontPath)
 		{
-			return AddFontResourceEx (fontPath, 0x10 /* FR_PRIVATE */, System.IntPtr.Zero) > 0;

```

Comment:
Could ```AddFontResourceEx``` still be relevant when embedded in some other application (e.g. for Xamarin inside VS)?
(cc @garuma)